gc/base/standard/ConcurrentGCSATB.cpp

```diff
@@ -474,6 +484,13 @@ MM_ConcurrentGCSATB::completeConcurrentTracing(MM_EnvironmentBase *env, uintptr_
 	}
 
 	Assert_MM_true(_markingScheme->getWorkPackets()->isAllPacketsEmpty());
+
+	TRIGGER_J9HOOK_MM_PRIVATE_GLOBAL_GC_INCREMENT_START(
```

Comment:
TRIGGER_J9HOOK_MM_PRIVATE_CONCURRENT_SATB_TOGGLED 
and as for the enable, move closer to preserveGlobalFragmentIndex, and wrap it up in disableSATB